src/Modules/ModParser.lua

```diff
@@ -1165,7 +1165,7 @@ local modTagList = {
 	["while in sand stance"] = { tag = { type = "Condition", var = "SandStance" } },
 	["while you have a bestial minion"] = { tag = { type = "Condition", var = "HaveBestialMinion" } },
 	["while you have infusion"] = { tag = { type = "Condition", var = "InfusionActive" } },
-	["while focussed"] = { tag = { type = "Condition", var = "Focused" } },
+	["while focus?sed"] = { tag = { type = "Condition", var = "Focused" } },
```

Comment:
Hey, I'm trying to find my way around the codebase and was wondering about this change. What is the reason for this `?` notation?

Reply:
The string on the left hand side is a Lua _pattern_ where the `?` denotes that the character/class preceding it is optional. This construct allows our code to keep matching both the old and new spelling. See <https://www.lua.org/pil/20.2.html> for info on patterns.

Reply:
@zao Awesome, thank you very much!